migrations/000017_create_namespace_recommendation_sets.up.sql

```diff
@@ -0,0 +1,23 @@
+CREATE TABLE IF NOT EXISTS namespace_recommendation_sets(
+   id uuid DEFAULT gen_random_uuid() PRIMARY KEY,
+   workload_id BIGINT NOT NULL,
+   namespace_name TEXT NOT NULL,
+   cpu_request_current FLOAT,
+   cpu_variation FLOAT,
+   memory_request_current FLOAT,
+   memory_variation FLOAT,
+   monitoring_start_time TIMESTAMP WITH TIME ZONE NOT NULL,
+   monitoring_end_time TIMESTAMP WITH TIME ZONE NOT NULL,
+   recommendations jsonb NOT NULL,
```

Comment:
Does that mean when the ROS OCP service receives metrics data that's insufficient to generate namespace-level recommendations, it will skip creating an entry in the table?

Reply:
The recommendations json is a required column to create a the recomm record.
This json is available only if Kruize sends the same to us, one of those reason would be insufficient metrics, etc.